force-app/main/default/classes/JobApplicationTriggerHandler.cls

```diff
@@ -1,5 +1,17 @@
+/**
+ * @description       : Operations on the JobApplication Object
+ * @author            : @William, @Tyler @Fritz
+ * @group             : Cohort2
+ * @last modified on  : 03-15-2024
+ * @last modified by  : @Fritz
+**/
+
 public with sharing class JobApplicationTriggerHandler {
 
+    public static final double MEDICARE_TAX_RATE = 0.0620;
```

Comment:
```suggestion
    private static final double MEDICARE_TAX_RATE = 0.0620;
```

Reply:
Same as the method mentioned, I feel these should be private unless you are pulling them elsewhere